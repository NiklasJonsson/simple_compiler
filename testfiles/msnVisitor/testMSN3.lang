int fun() {
	if(anton == bast) {
		while(nicke == nyfiken) {
			int hej;
		}
	}
}