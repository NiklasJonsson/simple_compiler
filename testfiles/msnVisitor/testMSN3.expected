Found Max Depth: 3
